### DISCUSSION TITLE
TSUN TSOL M-800 (DE) support confirmed

### DISCUSSION THREAD

**Discussion by bigcookie** [tbnobody/OpenDTU#422] [Show and tell]
Hi,

I wanted to confirm, that openDTU works fine with a TSUN TSOL M-800 (DE) version which is limited to 600W output.
Firmware version: 1.0.10
No problems whatsoever. Wanted to share, as the documentation states "firmware dependent". Probably this helps for improving the documentation :-).

Thanks for the great software!!!!! 

Edit: Serial number starts with 11

**Comment by homeautomation2022** [comment]
@tbnobody Kannst du bitte auf der Readme anpassen, dass es nicht wirklich Firmware abhängig ist(im Hintergrund ist das zwar richtig, aber die kennt ja keiner von außen), sondern Seriennummerabhängig ist?
11... = HM -> funktionell
10... = MI -> (noch) nicht mit OpenDTU kompatibel

Ich bin mir sicher das würde helfen, schon von vorn herein zu erkennen, ob man es überhaupt mit seinem Wechselrichter versucht.

**Reply by bigcookie** [reply]
I took the liberty to create a PR with a proposed update of the Readme.md based on this discussion: https://github.com/tbnobody/OpenDTU/pull/423

**Comment by itschaaf** [comment]
I can also confirm that the TSUN TSOL M-800(DE) works with OpenDTU with Firmware Version 1.0.10.
Also the TSUN TSOL M-1600 with Firmware Version 1.0.12 works with OpenDTU.

Edit: Both serial numbers starts with 11

**Comment by Remusplus** [comment]
I can confirm that the TSUN TSOL M-800(DE) works with OpenDTU with Firmware Version 1.0.10.
serial numbers starts with 11.

**Comment by cod3-joe** [comment]
I can't enter the serial number of my TSUN TSOL M-800(DE) in OpenDTU, because it contains letters. If I copy/paste the serial number, all letters are automatically removed. Afterwards only "Unknown" is displayed in the inverter list under "Type". The inverter was purchased at the end of March 2023.

The serial number starts with R17E730... After that 9 more characters follow.

Does anyone have any idea?

**Reply by DPO99** [reply]
Please take care:

_TSUN compatibility remark: Compatibility with OpenDTU is most likly related to the serial number of the inverter. Current findings indicate that TSUN inverters with a serial number starting with "11" are supported, whereby inverters with a serial number starting with "10" are not._

In this case your serial number does not start with 11....so i think its not supported.

**Comment by FlyingSpur2025** [comment]
I have the same problem with the serial number.
The SN starts with R17E7....
does anyone have any idea how to config this TSUN TSOL M-800(DE)?

**Comment by benderl** [comment]
I got the same type of inverter today. S/N is R17E730...
So far i realized that they might have integrated the DTU stick as the inverter can directly be connected to wifi network. The inverter can be configured with a app via bluetooth. Unfortunately I was not able to create an account with this fuxxxxg app for further investigation over wifi.

Does anybody know if a connection with openDTU is still possible? From my current point of view, I think they switched from DTU to wifi/bluetooth only.

**Comment by benderl** [comment]
I started a new discussion for devices with S/N R17E... here: https://github.com/tbnobody/OpenDTU/discussions/855
